etna/search/tests/test_search_tags.py

```diff
@@ -1,6 +1,10 @@
 from django.test import RequestFactory, SimpleTestCase
 
-from ..templatetags.search_tags import query_string_exclude, query_string_include
+from ..templatetags.search_tags import (
+    extended_in_operator,
+    query_string_exclude,
+    query_string_include,
+)
 
 
 class QueryStringTest(SimpleTestCase):
```

Comment:
Can we have a bit more descriptive title, like what the test is actually testing?

Reply:
This comment is seemed to be linked with the one below - ref `Like the above note please` - please see response in that location.